flake-modules/dev/list-plugins.py

```diff
@@ -67,9 +67,7 @@ def __str__(self) -> str:
                 assert False
         deprecation_icon: str = "⚠️" if self.dep_warnings else "  "
 
-        return (
-            f"| {kind_icon}\033[0m  | {state_icon}  | {deprecation_icon} | {self.path}"
-        )
+        return f"| {kind_icon}\033[0m  | {state_icon}  | {deprecation_icon} | {self.path}"
```

Comment:
Think this was committed by mistake?

Treefmt is complaining anyway.

Reply:
Maybe your copy of `ruff` is different or you have some config file we don't? Did you use `nix fmt`?

Reply:
FYI: I've re-pushed without the formatting commit